ipapython/ipautil.py

```diff
@@ -127,12 +127,34 @@ def __init__(self, addr):
         super(UnsafeIPAddress, self).__init__(addr,
                                               flags=self.netaddr_ip_flags)
 
+    def __getstate__(self):
+        state = {attr: getattr(self, attr) for attr in ['_net']}
```

Comment:
IMHO `state = {'_net': self._net}` is all you need, there is no benefit in dict comprehension.